addons/mail/static/src/models/country/country.js

```diff
@@ -19,9 +19,27 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        /**
+         * @private
+         * @returns {string|undefined}
+         */
+        _computeFlagUrl() {
+            if (!this.code) {
+                return;
```

Comment:
This should return `clear()` from `const { clear } = require('mail/static/src/model/model_field_command.js');` if we intent to actually remove the value if the `code` changes to falsy, though it's probably not a real use case (`code` should be considered required and read-only for most intents and purposes)

Reply:
Most of the flow don't give the country code so I don't think it's a good idea to make it required

Reply:
It was just conceptual comment, we don't have required yet anyway.